### PR TITLE
Flatten out the "..." to not hide assignment/exam actions

### PR DESCRIPTION
**Pull request by te-sa**
## Description

Turned the "..." drop-down menu on the assignments/exams table into three icons.

## Types of changes

What types of changes does your code introduce?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (only improve the code quality no change in functionality)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] Issue has been created
- [x] Every file touched is linted
- [ ] Every file touched has tests that cover all the funcitonality with green coverage
- [ ] Documentation is updated (if applicable).

## Issue

#54 

## Tests

No tests were added or changed.

## Screenshots (if appropriate):
Teacher view:
Screen Shot 2023-02-16 at 8 20 52 PM

Student view:
Screen Shot 2023-02-18 at 12 45 24 PM

### CODE REVIEW

File: src/app/course/event/event-row/event-row.component.html [hunk 1]
```diff
@@ -53,55 +53,69 @@
...
+            tuiDescribedBy="edit"
+            tuiHintDirection="top-left"
+            tuiHintId="edit"
+            tuiHintMode="onDark"
+            tuiIconButton
+        ></a>
+        <ng-template #editTooltip>
+            Edit
+        </ng-template>
+        <a
```
Comment (keyvankhademi) [inline src/app/course/event/event-row/event-row.component.html]: Please either use either `a` or `button` tag consistently for all of them.

File: src/app/course/event/event-row/event-row.component.html [hunk 2]
```diff
@@ -53,55 +53,69 @@
...
-                    (click)="setFeatured()"
-                    tuiOption
-                    [disabled]="event.featured"
-                    aria-label="Set Featured"
-                >
-                    Set Featured
-                </a>
-            </tui-data-list>
+        </ng-template>
+        <a
```
Comment (keyvankhademi) [inline src/app/course/event/event-row/event-row.component.html]: Let's have a full view button for the student view.

File: src/app/course/event/event-row/event-row.component.html [hunk 3]
```diff
@@ -53,55 +53,69 @@
             Grade: {{ event.total_event_grade | number: '1.0-1' || 'TBA'}}%
         </ng-template>
     </div>
-    <div class="flex items-center">
+    <div class="questions-table__actions" *ngIf="event.has_edit_permission">
```
Comment (keyvankhademi) [inline src/app/course/event/event-row/event-row.component.html]: The view button should not be under this permission. Please check the student view and make sure it works.

File: src/app/course/event/event-row/event-row.component.html [hunk 4]
```diff
@@ -4,10 +4,10 @@
         </ng-container>
         <ng-container *ngIf="event.type === 'EXAM'">Exam</ng-container>
     </div>
-    <div class="flex flex-grow items-center">
+    <div class="flex items-center sm:flex-grow lg:min-w-[21rem]">
```
Comment (keyvankhademi) [inline src/app/course/event/event-row/event-row.component.html]: I don't understand, why did you have to this?

**Review by keyvankhademi — CHANGES_REQUESTED** [review]

File: src/app/course/event/event-row/event-row.component.html (hunk 4 above)
Comment (te-sa) [reply src/app/course/event/event-row/event-row.component.html]: This is so the view doesn't look weird on mobile

**Review by keyvankhademi — APPROVED** [review]

**Pull request merged**